libs/core/hf2.cpp

```diff
@@ -8,6 +8,74 @@
 
 #if CONFIG_ENABLED(DEVICE_USB)
 
+static const char hidDescriptor[] = {
+    0x06, 0x97, 0xFF, // usage page vendor 0x97 (usage 0xff97 0x0001)
```

Comment:
@pelikhan please use `usage 0xff97 0x0001` in addition to the existing `usage 0xff00 0x0001` in the winrt app - this matches the bootloader. Once we think all bootloaders are updated, we can remove the `0xff00`.